setup.py

```diff
@@ -37,7 +37,7 @@ def get_static_files(path):
 with open('README.rst') as fobj:
     long_description = "\n\n" + readme_note + "\n\n" + fobj.read()
 
-install_requires = ['python-dateutil>=2.7.5,<3', 'tenacity>=6.3.0,<7']
+install_requires = ['python-dateutil>=2.7.5,<3', 'tenacity>=6.3.0,<7', 'kubernetes==17.17.0']
```

Comment:
given that the Kubernetes job type is an optional portion of Luigi, can we not make `kubernetes` package install required when installing `luigi`?

Reply:
Yep, I'll happily realign to this if it's optional.  Will do in some upcoming free time.  Thanks for the feedback.

Reply:
Resolved in latest push